### DISCUSSION TITLE
187

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#159] [General]
# 潮流周刊第187期 - 我想开了

封面图拍摄于周末去杭州葛巷地铁站 D 口停车场后面一大片荷花地拍的照片，夕阳西下时分，光照效果很棒，拍到了有我自闭了、我快想开了，我想开了这几种荷花，哈哈这里放一朵快想开的荷花。

https://weekly.tw93.fun/posts/187-%E6%88%91%E6%83%B3%E5%BC%80%E4%BA%86

**Comment by Vainycos** [comment]
最后的生活哲学很有用

**Comment by wukaipeng-dev** [comment]
好像没射中，仔细看箭火擦过了，点燃是程序控制的

**Comment by BaoXuebin** [comment]
> 在炒菜过程中可以把盛菜的盘子和灶台给收拾了

一直这么做的，因为吃饱了就不想动了 ;-|

**Reply by tw93** [reply]
哈哈 是一个很好的习惯
